protoc-gen-connect-kotlin/src/main/kotlin/build/buf/protocgen/connect/Generator.kt

```diff
@@ -46,27 +48,20 @@ import com.squareup.kotlinpoet.asClassName
  * move off of type aliases, this can be changed without user API breakage.
  */
 private val HEADERS_CLASS_NAME = ClassName.bestGuess("build.buf.connect.Headers")
```

Comment:
FYI, can also do `ClassName.get("build.buf.connect", "Headers")` to directly indicate the package and the class name (instead of making `bestGuess` figure it out via string parsing).